clients/client-python/gravitino/client/gravitino_metalake.py

```diff
@@ -185,19 +185,47 @@ def alter_catalog(self, name: str, *changes: CatalogChange) -> Catalog:
             self.name(), catalog_response.catalog(), self.rest_client
         )
 
-    def drop_catalog(self, name: str) -> bool:
+    def drop_catalog(self, name: str, force: bool = False) -> bool:
         """Drop the catalog with specified name.
 
         Args:
-            name the name of the catalog.
+            name: the name of the catalog.
+            force: whether to force drop the catalog.
 
         Returns:
-            true if the catalog is dropped successfully, false otherwise.
+            true if the catalog is dropped successfully, false if the catalog does not exist.
         """
+        params = {"force": str(force)}
```

Comment:
`str(force)` will get "True/False" or "true/false"?

Reply:
In any case, the server side will ignore case sensitivity during query param conversion. So it's not a problem